packages/plugins/typescript/resolvers/src/index.ts

```diff
@@ -76,7 +76,7 @@ export type NewStitchingResolver<TResult, TParent, TContext, TArgs> = {
 
     defsToInclude.push(`
       type ScalarCheck<T, S> = S extends true ? T : NullableCheck<T, S>;
-      type NullableCheck<T, S> = Maybe<T> extends T ? Maybe<ListCheck<NonNullable<T>, S>> : ListCheck<T, S>;
+      type NullableCheck<T, S> = ${namespacedImportPrefix}Maybe<T> extends T ? ${namespacedImportPrefix}Maybe<ListCheck<NonNullable<T>, S>> : ListCheck<T, S>;
```

Comment:
I believe this is correct, but I'm not sure how to test it and it doesn't look like it's currently covered by any tests. I'd be happy to add a test, if someone can give me direction. Otherwise, I can reverse this change if it could be a problem.